src/power_grid_model_io/cli/convert.py

```diff
@@ -0,0 +1,104 @@
+# SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model IO project <dynamic.grid.calculation@alliander.com>
+#
+# SPDX-License-Identifier: MPL-2.0
+"""
+Command Line Interface to convert files from one type to another
+"""
+
+# pylint: disable=duplicate-code
+
+import logging
+from pathlib import Path
+from typing import Optional
+
+import structlog
+from power_grid_model.validation import errors_to_string, validate_input_data
+
+from power_grid_model_io.converters.gaia_excel_converter import GaiaExcelConverter
+from power_grid_model_io.converters.pgm_converter import PgmConverter
+from power_grid_model_io.converters.vision_excel_converter import VisionExcelConverter
+from power_grid_model_io.utils.modules import import_optional_module
+
+typer = import_optional_module("cli", "typer")
```

Comment:
I'd prefer to specify the parameters here, e.g. `typer = import_optional_module(extra="cli", module="typer")`

Reply:
Yes, I agree. I'll fix it.